terra_layer/migrations/0047_auto_20191211_1016.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.7 on 2019-12-11 10:16
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("terra_layer", "0046_auto_20191210_1437"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(name="layergroup", unique_together=set(),),
+    ]
```

Comment:
Can you squash all migrations before merging?

Reply:
Yes a will do it later in some time after this message :D